src/Components/Article/Article.jsx

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import styles from '../Article/Article.module.css'
+
+function Article() {
+    const [articles, setAllArticles] = useState([
+        {
+            article_id: '',
+            author: '',
+            body: '',
+            comment_count: '',
+            created_at: '',
+            title: '',
+            topic: '',
+            votes: '',
+        },
+    ])
+
+    const [isThereAnyArticle, setIsThereAnyArticle] = useState(true)
+
```

Comment:
like the conversation we had last week this is something you don't need state for

you can derive this information from `articles.length` - it's not something that needs to be stored in state!